.ko.yaml

```diff
@@ -155,6 +154,91 @@ builds:
       org.opencontainers.image.revision: "{{.Env.GIT_COMMIT}}"
       org.opencontainers.image.created: "{{.Env.BUILD_DATE}}"
 
+  - id: janitor-provider-aws
```

Comment:
Instead of making one per CSP, can we make a single provider and switch based on env or something? This will result in an explosion of container images as we scale :)